Update software_templates.md (#37866)
DataDog/documentation: content/en/internal_developer_portal/self_service_actions/software_templates.md

Example blueprints:

> - **[Scaffold New Project in GitHub blueprint][11]**: Scaffolder app can be used for creating new software components from template repositories. It takes inputs from developers and generates a new repository or a PR based on the provided data and the template.

Reviewer: ```suggestion
- **[Scaffold New Project in GitHub blueprint][11]**: Create new repositories or pull requests from a template repository using developer-provided inputs.
```

Rewording to avoid passive voice and make this more consistent with the other examples in the list. You could also probably drop the last clause ("using developer-provided inputs").

Reviewer: I noticed that this same text appears in the UI. Outside of the scope of this PR, but I would recommend updating it there to match:

<img width="452" height="215" alt="image" src="https://github.com/user-attachments/assets/accce8ef-81a1-43b1-bf97-66d5c08df32c" />
